### DISCUSSION TITLE
Please return changelog links to release descriptions

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#23107] [Ideas]
I have been subscribed to email notifications about releases for a while. Previously, emails (release descriptions) used to contain a link to the release's specific changelog on the [changelog site](https://osu.ppy.sh/home/changelog), which had nice formatting and embeds. For the last few releases, that has been replaced with including the changelogs in descriptions, which is harder to read because of the barebones, markdown formatting.
Please return the changelog links to the osu! website to descriptions.

**Comment by peppy** [comment]
You can bookmark the changelog site for quick access. Or access it from the game itself (it shows after an update).

The github formatted changelogs are nothing new, the only thing missing is a link. Might bring it back when we re-automated the flow, we'll see.
